components/home/IntegrationDestinations.tsx

```diff
@@ -20,8 +20,8 @@ export default function IntegrationDestinations() {
                     key={`integrationDestination-${idx}`}
                     lg={4}
                     md={4}
-                    sm={6}
-                    xs={6}
+                    sm={12}
+                    xs={12}
```

Comment:
Now that we named things `Mailchimp Integration` rather than just `Mailchimp` the buttons take up more space